packages/amplify-cli/src/init-steps/s0-analyzeProject.ts

```diff
@@ -297,3 +299,29 @@ function getDefaultEditor() {
 
   return localEnvInfo.defaultEditor;
 }
+
+/**
+ *Checks if `amplify env add` has been executed, and extract environment name if exists
+ * @param {$TSContext} context The Amplify context object
+ * @returns `{exec: boolean, envName?: string | null}`
+ *
+ * `exec` denotes if `amplify env add` has been executed
+ *
+ * `envName` extracts environment name if provided
+ */
+function addNewEnv(context): { exec: boolean; envName?: string | null } {
```

Comment:
Can you please give this a different name. It currently seems like this function would add a new environment.

Reply:
Resolved!